hikari/config.py

```diff
@@ -330,10 +330,10 @@ def _(self, _: attr.Attribute[typing.Optional[int]], value: typing.Optional[int]
         if value is not None and (not isinstance(value, int) or value <= 0):
             raise ValueError("http_settings.max_redirects must be None or a POSITIVE integer")
 
-    _ssl: typing.Union[bool, ssl_.SSLContext] = attr.field(
-        default=True,
+    _ssl: ssl_.SSLContext = attr.field(
+        factory=lambda: _ssl_factory(True),
         converter=_ssl_factory,
-        validator=attr.validators.instance_of(ssl_.SSLContext),  # type: ignore[assignment,arg-type]
+        validator=attr.validators.instance_of(ssl_.SSLContext),
     )
```

Comment:
Since we are here, could you merge this with the `ssl` property. There is no need for it anymore :)

Reply:
done